docker-compose-dev.yml

```diff
@@ -0,0 +1,399 @@
+version: "3"
```

Comment:
We don't need a `docker-compose-dev.yml`. Dev will be merged into the main.

Reply:
Sure! Will remove this.